cd /t3home/fglessge/CMSSW_10_1_0/src/Xbb/python/
Task: sysnew
Pwd: /t3home/fglessge/CMSSW_10_1_0/src/Xbb/python
Host: t3wn52
User: fglessge
StartTime: 2022-02-15 12:46:55

Parsing files in Zll2017config...

python ./sys_new.py --addCollections Sys.all --fileList base64:eNolx7FuwyAQgOEncu+O5sCZ2yGq7CiDZakT4gw2kVKDAA95+0jpv3z6obZUAmwlHRlyfFYb79tWIYrA3o78CBXmi8gt1XZ1ewKFZGAmgu/fn9DqlIbBjh2jvUydRmypR7TTsYevG1v6nMLc/Tm/FZfjOIxdfrZ4dz2syvXOoz+JIlZ6oSBnx9qQrJqFFZmg5Wx6XuTkF09sVi1MCPjO/vM+aCUEqz5KSu0FxGpAMg== --sampleIdentifier DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8 --config /t3home/fglessge/CMSSW_10_1_0/src//Xbb/python/logs_Zll2017//sysnew-sys/config//submission_20220215_124524.ini --inputDir SYSin --outputDir SYSout
Error in <TNetXNGFile::Open>: [ERROR] Server responded with an error: [3011] No such file

len(filelist) 1 filelist[0]: /store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2017/V11/DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510/000000_000000/0000/tree_2.root
INput samples:	root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd/
OUTput samples:	root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/sys/
collections to add: ['Sys.all']
after parsing: ['VHbbCommon.SampleGroup', 'VHbbCommon.BTagSF', 'Sys.electronSF', 'Sys.muonSF', 'Sys.Run2017BmuHLT', 'VHbbCommon.HiggsReco', 'VHbbCommon.vLeptons', 'VHbbCommon.EWKweights', 'VHbbCommon.DYspecialWeight', 'VHbbCommon.LOtoNLOweightV7', 'VHbbCommon.JetPUIDSF', 'VHbbCommon.DoubleBtagSF']
INFO: reading part  1  of  1
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_USER_PROXY = /t3home/fglessge/.x509up_u742
INFO: file was not found:root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd//DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/tree_f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510_000000_000000_0000_2_bb40ecdcff8f5f3eca9c4f7e42c7bcb93e6fe793917f73c93a052b29.root, trying 2 more times...
INFO: wait 1 seconds before trying again
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_USER_PROXY = /t3home/fglessge/.x509up_u742
INFO: file was not found:root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd//DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/tree_f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510_000000_000000_0000_2_bb40ecdcff8f5f3eca9c4f7e42c7bcb93e6fe793917f73c93a052b29.root, trying 1 more times...
INFO: wait 2 seconds before trying again
[31mERROR: file is missing: root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd//DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/tree_f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510_000000_000000_0000_2_bb40ecdcff8f5f3eca9c4f7e42c7bcb93e6fe793917f73c93a052b29.root[0m
INFO: # files chained: 0
trying fallback... 1
FO: root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd///store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2017/V11/DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510/000000_000000/0000/tree_2.root
INFO: reading part  1  of  1
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_USER_PROXY = /t3home/fglessge/.x509up_u742
INFO: file was not found:root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd///store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2017/V11/DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510/000000_000000/0000/tree_2.root, trying 2 more times...
INFO: wait 1 seconds before trying again
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_USER_PROXY = /t3home/fglessge/.x509up_u742
INFO: file was not found:root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd///store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2017/V11/DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510/000000_000000/0000/tree_2.root, trying 1 more times...
INFO: wait 2 seconds before trying again
[31mERROR: file is missing: root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd///store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2017/V11/DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510/000000_000000/0000/tree_2.root[0m
INFO: # files chained: 0
[31mERROR: file does not exist or is broken, will be SKIPPED![0m
Traceback (most recent call last):
  File "./sys_new.py", line 301, in <module>
    xr.run()
  File "./sys_new.py", line 281, in run
    raise Exception("ProcessingIncomplete")
Exception: ProcessingIncomplete
--------------------------------------------------------------------------------
exit code: 1
EndTime: 2022-02-15 12:47:17
duration (real time): 0 minutes
--- RETRY ---
python ./sys_new.py --addCollections Sys.all --fileList base64:eNolx7FuwyAQgOEncu+O5sCZ2yGq7CiDZakT4gw2kVKDAA95+0jpv3z6obZUAmwlHRlyfFYb79tWIYrA3o78CBXmi8gt1XZ1ewKFZGAmgu/fn9DqlIbBjh2jvUydRmypR7TTsYevG1v6nMLc/Tm/FZfjOIxdfrZ4dz2syvXOoz+JIlZ6oSBnx9qQrJqFFZmg5Wx6XuTkF09sVi1MCPjO/vM+aCUEqz5KSu0FxGpAMg== --sampleIdentifier DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8 --config /t3home/fglessge/CMSSW_10_1_0/src//Xbb/python/logs_Zll2017//sysnew-sys/config//submission_20220215_124524.ini --inputDir SYSin --outputDir SYSout
Error in <TNetXNGFile::Open>: [ERROR] Server responded with an error: [3011] No such file

len(filelist) 1 filelist[0]: /store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2017/V11/DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510/000000_000000/0000/tree_2.root
INput samples:	root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd/
OUTput samples:	root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/sys/
collections to add: ['Sys.all']
after parsing: ['VHbbCommon.SampleGroup', 'VHbbCommon.BTagSF', 'Sys.electronSF', 'Sys.muonSF', 'Sys.Run2017BmuHLT', 'VHbbCommon.HiggsReco', 'VHbbCommon.vLeptons', 'VHbbCommon.EWKweights', 'VHbbCommon.DYspecialWeight', 'VHbbCommon.LOtoNLOweightV7', 'VHbbCommon.JetPUIDSF', 'VHbbCommon.DoubleBtagSF']
INFO: reading part  1  of  1
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_USER_PROXY = /t3home/fglessge/.x509up_u742
INFO: file was not found:root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd//DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/tree_f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510_000000_000000_0000_2_bb40ecdcff8f5f3eca9c4f7e42c7bcb93e6fe793917f73c93a052b29.root, trying 2 more times...
INFO: wait 1 seconds before trying again
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_USER_PROXY = /t3home/fglessge/.x509up_u742
INFO: file was not found:root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd//DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/tree_f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510_000000_000000_0000_2_bb40ecdcff8f5f3eca9c4f7e42c7bcb93e6fe793917f73c93a052b29.root, trying 1 more times...
INFO: wait 2 seconds before trying again
[31mERROR: file is missing: root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd//DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/tree_f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510_000000_000000_0000_2_bb40ecdcff8f5f3eca9c4f7e42c7bcb93e6fe793917f73c93a052b29.root[0m
INFO: # files chained: 0
trying fallback... 1
FO: root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd///store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2017/V11/DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510/000000_000000/0000/tree_2.root
INFO: reading part  1  of  1
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_USER_PROXY = /t3home/fglessge/.x509up_u742
INFO: file was not found:root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd///store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2017/V11/DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510/000000_000000/0000/tree_2.root, trying 2 more times...
INFO: wait 1 seconds before trying again
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_VOMS_DIR = /etc/grid-security/vomsdir
DEBUG: X509_USER_PROXY = /t3home/fglessge/.x509up_u742
INFO: file was not found:root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd///store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2017/V11/DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510/000000_000000/0000/tree_2.root, trying 1 more times...
INFO: wait 2 seconds before trying again
[31mERROR: file is missing: root://t3dcachedb03.psi.ch:1094//pnfs/psi.ch/cms/trivcat/store/user/fglessge/VHbb/Zll/VHbbPostNano2017_V11/2020_06_08/hadd///store/group/phys_higgs/hbb/ntuples/VHbbPostNano/2017/V11/DYJetsToLL_M-50_HT-600to800_TuneCP5_13TeV-madgraphMLM-pythia8/f2a8ad0d4b21526c1eb9a5671bf65b5217e6b9785cb4dcd157f6b510/000000_000000/0000/tree_2.root[0m
INFO: # files chained: 0
[31mERROR: file does not exist or is broken, will be SKIPPED![0m
Traceback (most recent call last):
  File "./sys_new.py", line 301, in <module>
    xr.run()
  File "./sys_new.py", line 281, in run
    raise Exception("ProcessingIncomplete")
Exception: ProcessingIncomplete
exit code: 1
duration (real time): 0 minutes, including retry

Exiting runAll.sh

